packages/main/src/Input.js

```diff
@@ -1067,6 +1067,7 @@ class Input extends UI5Element {
 
 		return {
 			"input": {
+				"ariaRoledescription": this._inputAccInfo && this._inputAccInfo.ariaRoledescription || undefined,
```

Comment:
There is an eslint rule enforcing () despite priority for this line.

If you want, you can create a git pre push hook with the following content to avoid pushing eslint problems:
```
#!/bin/sh
#
# To enable this hook, run this command from the root folder of the project
#   $ git config core.hooksPath .githooks

# git runs hooks in the root dir
yarn lint
```

![image](https://user-images.githubusercontent.com/15844574/104305186-098dfe80-54d5-11eb-962d-87295e2e5a34.png)